timescaledb/how-to-guides/analytics/about-analytics.md

```diff
@@ -0,0 +1,75 @@
+# About Timescale Analytics
+Timescale Analytics is a PostreSQL extension containing a specialized set of
+functions that allow you to to analyze time-series data. You can use it to
+analyze anything you have stored as time-series data, including IoT devices, IT
+systems, marketing analytics, user behavior, financial metrics, and
+cryptocurrency. Timescale Analytics uses [Promscale][doc-promscale], an
```

Comment:
It doesn't...I think this is out of date, but some of this copy will be better taken from new introductory blog post rather than the old...

Reply:
Which bit is out of date?